base/server/python/pki/server/__init__.py

```diff
@@ -540,9 +540,21 @@ def get_startup_tests(self):
 
         return target_tests
 
-    def set_startup_tests(self, target_tests):
-        # Remove unnecessary space, curly braces
-        self.config['selftests.container.order.startup'] = ", "\
+    def set_startup_tests(self, critical, test=None):
```

Comment:
Previously the method could be used to update multiple tests with different critical values at once. Now it's limited to update all tests or just a single test with a single critical value.

I think the original method should be preserved, and if you need to update just one test it can be done like this:
~~~
subsys.set_startup_tests([{test: True}])
~~~
or create a separate method that updates just one test:
~~~
subsys.set_startup_test(name=test, critical=True)
~~~